tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +191,33 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_with_added_user(self):
+        group_name = "TestGroupName"
+        response = self.client.post(
+            reverse("admin:auth_group_add"),
+            {"name": group_name, "users": [self.tester.id]},
+            follow=True,
+        )
+
+        self.assertContains(response, group_name)
+        self.assertContains(response, "was added successfully.")
```

Comment:
This fails for French b/c you need to match a translated string. However this is probably a part of a larger string which includes placeholders. You'll have to search for it in Django's source code.